Autopilot/PathManager/Inc/trackFinder.hpp

```diff
@@ -1,7 +1,9 @@
 #pragma once
```

Comment:
THis file does not need to exist. Track is just pulled from the sensor fusion output. The guts of this file should live inside SensorFusion.cpp

Reply:
I don't think sensorFusion has those changes yet though?